backend/curriculum_tracking/tests/test_UserStatsPerWeekSerializer.py

```diff
@@ -0,0 +1,234 @@
+        )
+        self.pull_request.save()
+
+    def test_one_card_in_progress_before_any_reviews(self):
+
+        # One card, no reviews just yet, function 'get_cards_in_progress_column' should return one card
+        assert self.card_1.status == AgileCard.IN_PROGRESS
+        assert self.project_1.content_item == self.card_1.content_item
+        assert AgileCard.derive_status_from_project(self.project_1) == AgileCard.IN_PROGRESS
+        assert UserStatsPerWeekSerializer.get_cards_in_progress_column(UserStatsPerWeekSerializer, User=self.user) == 1
+
+    def test_create_four_reviews_cards_will_be_in_different_columns(self):
+
+        # Request for a review to happen on project_1, card_1
+        request_review_time = self.project_1.start_time + timedelta(1)
+        self.project_1.request_review(force_timestamp=request_review_time)
+        time_one = self.project_1.start_time - timedelta(days=6)
+        time_two = self.project_1.start_time + timedelta(days=4)
+        time_three = self.project_1.start_time + timedelta(days=3)
+
+        # Three reviews are made at different times, so we should have different cards in different columns depending
+        # on the outcome of the review
+        review_1 = RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=time_one
+        )
+        review_1.timestamp = time_one
+        review_1.save()
+
+        review_2 = RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = time_two
+        review_2.save()
+
+        review_3 = RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = time_three
+        review_3.save()
+
+        # review_1 had a status of NYC, so we should have at least one card in the 'REVIEW FEEDBACK' column
+        assert UserStatsPerWeekSerializer.get_cards_in_review_feedback_column(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 1
+
+        # review_2 had a status of COMPETENT, the reviewer is not trusted and therefore we should have zero cards in
+        # the completed column
+        assert UserStatsPerWeekSerializer.get_cards_in_completed_column(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 0
+
+        # review_3 had a status of Excellent, the reviewer is not trusted and therefore we should have zero cards in
+        # the completed column
+        assert UserStatsPerWeekSerializer.get_cards_in_completed_column(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 0
+
+        # In total we did start new cards within the last 7 days but we did not complete any cards in the last seven
+        # days
+        assert UserStatsPerWeekSerializer.get_cards_started_last_7_days(
+            UserStatsPerWeekSerializer, User=self.user
+        ) > 0
```

Comment:
Can we be more specific? What exact number are we expecting here?

Most of the time it makes sense to assert specific things